src/Infrastructure/Domain/Doctrine/DoctrineSymfonyUserRepository.php

```diff
@@ -7,54 +7,46 @@
 use Becklyn\Security\Domain\UserNotFoundException;
 use Becklyn\Security\Infrastructure\Domain\Symfony\SymfonyUserRepository;
 use Doctrine\ORM\EntityManagerInterface;
-use Doctrine\ORM\EntityRepository;
 
 /**
  * @author Marko Vujnovic <mv@201created.de>
+ *
  * @since  2020-03-03
  */
 class DoctrineSymfonyUserRepository implements SymfonyUserRepository
 {
-    /**
-     * @var EntityManagerInterface
-     */
-    private $entityManager;
-
-    /**
-     * @var EntityRepository
-     */
-    private $repository;
+    private readonly \Doctrine\ORM\EntityRepository $repository;
```

Comment:
This class should be imported... like it already was but was apparently removed by cs-fixer.